### PR TITLE
Update taxpasta

### PR DESCRIPTION
**Pull request by sofstam**
This PR updates taxpasta module to 0.5.0

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by jfy133 — CHANGES_REQUESTED** [review]
- [ ] Need changelog depedency table update
- [ ] Need to add support for the new flags (rank lineage, for example)

**Comment by sofstam** [comment]
We already have support for:

```
    taxpasta_add_name                       = false
    taxpasta_add_rank                       = false
    taxpasta_add_lineage                    = false
    taxpasta_add_idlineage                  = false
```

Or do you mean something different?

**Comment by jfy133** [comment]
Sorry new flag (singular)

https://taxpasta.readthedocs.io/en/latest/quick_reference/standardise/

`--add-rank-lineage` as added in https://github.com/taxprofiler/taxpasta/pull/130

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#319](https://github.com/nf-core/taxprofiler/pull/319) Added support for virus hit expansion in Kaiju (❤️ to @dnlrxn for requesting, added by @jfy133)
 - [#323](https://github.com/nf-core/taxprofiler/pull/323) Add ability to skip sequencing quality control tools (❤️ to @vinisalazar for requesting, added by @jfy133)
 - [#345](https://github.com/nf-core/taxprofiler/pull/345) Add simple tutorial to explain how to get up and running with an nf-core/taxprofiler run (added by @jfy133)
+- [#355](https://github.com/nf-core/taxprofiler/pull/345) Add support for TAXPASTA's `--add-rank-lineage` to output (❤️ to @MajoroMask for request, added by @Midnighter, @sofstam, @jfy133)
```
Comment (jfy133) [inline CHANGELOG.md:21]:
```suggestion
- [#355](https://github.com/nf-core/taxprofiler/pull/355) Add support for TAXPASTA's `--add-rank-lineage` to output (❤️ to @MajoroMask for request, added by @Midnighter, @sofstam, @jfy133)
```

File: nextflow_schema.json [hunk 2]
```diff
@@ -615,25 +615,31 @@
...
+                    "help_text": "\nThe standard output format of taxpasta is a two-column table including the read counts and the integer taxonomic ID. The taxon's entire taxonomic lineage with the taxon identifiers separated by semi-colons can be added as additional information to the output table.\n\nModifies tool parameter(s):\n- taxpasta: `--add-id-lineage`\n",
+                    "fa_icon": "fas fa-link"
+                },
+                "taxpasta_add_ranklineage": {
+                    "type": "boolean",
+                    "description": "Add the taxon's entire rank lineage to the output.",
+                    "help_text": "\nThe standard output format of taxpasta is a two-column table including the read counts and the integer taxonomic ID. The taxonomicl levels taxon's entire lineage with the taxon identifiers separated by semi-colons can be added as additional information to the output table. This complements `--taxpasta_add_lineage` by telling you which taxonomic rank level each entry in the lineage refers to.\n\nModifies tool parameter(s):\n- taxpasta: `--add-rank-lineage`\n",
```
Comment (jfy133) [inline nextflow_schema.json:642]:
```suggestion
                    "help_text": "\nThe standard output format of taxpasta is a two-column table including the read counts and the integer taxonomic ID. The taxonomic ranks categories of the taxon's entire lineage separated by semi-colons can be added as additional information to the output table. This complements `--taxpasta_add_lineage` by telling you which taxonomic rank level each entry in the lineage refers to.\n\nModifies tool parameter(s):\n- taxpasta: `--add-rank-lineage`\n",
```

**Review by jfy133 — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -46,7 +47,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 | Tool      | Previous version | New version |
 | --------- | ---------------- | ----------- |
 | MultiQC   | 1.13             | 1.14        |
-| taxpasta  | 0.2.3            | 0.4.1       |
+| taxpasta  | 0.2.3            | 0.5.0       |
```
Comment (jfy133) [inline CHANGELOG.md:50]:
```suggestion
| TAXPASTA  | 0.2.3            | 0.5.0       |
```

**Pull request merged**